src/background.js

```diff
@@ -0,0 +1,38 @@
+function runShowMeTheCode(tabId, tab) {
+    function isGitHub(tab) {
```

Comment:
Actually - isn't tab in scope here? I think we can skip passing it to all `isXXX(tab)` functions